docs/modules/operation/pages/bsm/edges.adoc

```diff
@@ -5,36 +5,35 @@ Edges map the alarm status monitoring with OpenNMS.
+This will automatically provide edges for the nodeLostService, interfaceDown, and nodeDown reductions keys of the specified service.
+Reduction Key:: A resolved reduction key used to refer to a specific alarm.
+This must match a specific alarm reduction key including, not the reduction key formula from your event definitions.
+Application:: A reference to an existing application.
+This will automatically provide edges for the nodeLostService, interfaceDown, and nodeDown reductions keys of the defined services of this application.
 
 TIP: If you need help determining the reduction key used by alarm, trigger the alarm in question and pull the reduction key from the alarm details page.
 
 All edge types have the following parameters:
 
-* Map Function: The associated Map Function for this edge.
-* Weight: The relative Weight of this edge. Used by certain reduce functions.
+Map Function:: The associated xref:operation:bsm/map-functions.adoc[Map Function] for this edge.
+Weight:: The relative weight of this edge.
+Used by certain reduce functions to adjust how much the status of an edge impacts the business service status.
 
 Both IP Service and Reduction Key type edges also support a friendly name parameter which gives the user control on how the edge is labeled in the Topology User Interface.
-The editor changing the edge attributes is shown in figure <<ga-bsm-edge-edit, Editor to add Business Service Edges>>.
-
-[[ga-bsm-edge-edit]]
-.Editor to add Business Service Edges
-image::bsm/02_bsm-edge-edit.png[Business Service Edge edit,400,199]
 
 == Child Services
 
-To create a hierarchy of Business Services they need to be created first.
-The hierarchy is build by selecting the Business Service as Child Service as dependency.
+To create a hierarchy of Business Services, create the child service(s) first.
+The hierarchy is build by selecting the Business Service as Child Service dependency.
 
 == IP Services
 
-The IP Service is a predefined set of reduction keys that lets you easily assign a specific monitored service to the given BS.
-As an example you have multiple Servers with a monitored service SMTP and you want to model a BS named Mail Communication.
-If just the reduction key for a nodeLostService is assigned, the BS would not be affected in case the IP Interface or the whole Node goes down.
-OpenNMS generates alarms with different UEI which needs to be assigned to the BS as well.
+The IP Service is a predefined set of reduction keys that lets you easily assign a specific monitored service to the given business service.
+As an example, you have multiple Servers with a monitored service SMTP and you want to model a business service named Mail Communication.
+If just the reduction key for a nodeLostService is assigned, the business service would not be affected in case the IP Interface or the whole Node goes down.
+OpenNMS generates alarms with different UEI which needs to be assigned to the business service as well.
```

Comment:
is it the UEI or the alarms that need to be assigned to the business service? As it reads now, it's the UEI.

Reply:
BSM is based on reduction keys, which is a field on alarms, not the entire alarm object. After re-reading, this sentence isn't needed.
```suggestion
```